### ISSUE TITLE
Delayed controls

### ISSUE THREAD

**Issue by pikachu1001000** [sultim-t/Serious-Engine-RT#6]
On Nvidia 3070, the mouse controls are delayed when the ray tracing is enabled. However, if I switch to the Vulkan API, disabling the ray tracing, they are no longer delayed.

**Comment by sultim-t** [comment]
Apparently, the input controls are bound to the framerate, specifically in Serious Sam Classic. With Vulkan renderer, the `V-sync` is disabled by default, so most likely the game runs in 300FPS, so the delay isn't significant. With RT renderer, `V-sync` is enabled by default => FPS is bound to 60 (or other monitor refresh rate), and controls are less responsive.

The only suggestion is to enable `Options -> Players and Controls -> Sharp turning`.
Disabling `Vertical syncronization` might also help, but it will require more ray traced frames to be processed per second, which will cause GPU heating.

**Comment by Kibergx** [comment]
I have the same issue. Can't you setting in vulkan the pre renderes frames? There is same kind of option as far I know, in vulkan api the developer can control the pre rendered frames count. NVIDIA's control panel's ultra latency mode do nothing. :( This Sharp turning helped a lot, thank you. But still not so great I think I can't play in fullhd with rtx 2060 super.

**Comment by TimmyP777** [comment]
This cant be intended functionality. It is totally unplayable. My sensitivity is at 0% and it has input lag like I have never seen before.

**Comment by sultim-t** [comment]
Yes, it's truly uncomfortable to play with such delay. Sadly I don't have time (and PC) to fix it now. I really hope someone will investigate the possible solutions. Sorry :(

**Comment by phant** [comment]
> This cant be intended functionality. It is totally unplayable. My sensitivity is at 0% and it has input lag like I have never seen before.

 Sharp tuning = yes
 Mouse acceleration = no
 Sensitivity = 0%
 DLSS = Quality (to get 75-100 fps)

My settings for comfortable gameplay.

**Comment by LethalManBoob** [comment]
You are making sure that the gpu is not maxing out right?

**Comment by Calinou** [comment]
Out of curiosity, can you reproduce this with fullscreen disabled and/or a FPS cap set (`sam_iMaxFPSActive = 60` in console)?
